runtime/ts/relation.ts

```diff
@@ -15,14 +15,20 @@ export class Relation extends Entity {
   entities: Entity[];
 
   constructor(...entities) {
-    super();
+    super(undefined);
     this.entities = entities;
   }
 
+  // should be rawData???
```

Comment:
No, you can't have both a field and a getter with the same name. I'm pretty sure this should be `data`.

Reply:
Looking at the history of this code it appears that `data` was switched to `rawData` -- I was wondering if this was the case here.